assets/src/dashboard/app/views/myStories/index.js

```diff
@@ -54,10 +56,11 @@ const FilterContainer = styled.div`
   }
 
   @media ${({ theme }) => theme.breakpoint.min} {
-    & > label {
+    & > label span {
       border-radius: 0;
-      box-shadow: none;
+      box-shadow: none !important;
       padding: 0 10px 0 0;
+      background-color: transparent !important;
```

Comment:
Is there no way to do this without needing the `!important`?

Reply:
Not really right now? This is particularly rough because this override is just because we don't know what these filters should look like on small screens. I had initially thought that removing their 'pill' shape would be a better use of space. I don't know if this is a thing that will stick or if it's a thing that would be for all floating tab components when they are the minimum viewport size or just the dashboard filter section and if it's just the filter section perhaps that's it's own type of component and the set up can happen there? 

In anycase, the background color one is actually not necessary so i'm taking that out but the box shadow one is really hard to access without important because of styling the checked input nested in the label that also has a span. 

```
const FloatingTabLabel = styled(PillLabel)`
  ${PillInput}:checked + & {}`;
```